scss/_patterns_image-grid.scss

```diff
@@ -0,0 +1,15 @@
+// Image grid to showcase group of images
+@mixin vf-p-p-image-grid {
+
+  .p-p-image-grid {
+    @extend .clearfix;
+    text-align: center;
+    display: block;
+
+    &__img {
```

Comment:
I'd prefer to leave this light touch as responsibility for sourcing and using correctly sized images should be on the author.. by adding height/width stipulations, we leave the pattern more brittle and also open the possibility of people using huge images without realising...